Dockerfile

```diff
@@ -1,27 +1,42 @@
-FROM ubuntu:14.04
-MAINTAINER Shane Canon <scanon@lbl.gov>
+FROM eclipse-temurin:11.0.25_9-jdk-noble
```

Comment:
Since it's not there anymore, you might want to throw in a comment of what version of the OS this contains.

Reply:
Is `noble` in the tag not enough?

Reply:
I had no idea what that was until you said so. Didn't know it was Ubuntu vs. Debian vs. Alpine or whatever. I assumed that was part of the temurin image name.

Reply:
Fair enough, updated